orchagent/portsorch.cpp

```diff
@@ -3074,6 +3074,38 @@ bool PortsOrch::removeVlanHostIntf(Port vl)
     return true;
 }
 
+void PortsOrch::updateDbPortFlapCount(Port& port, sai_port_oper_status_t pstatus)
+{
+    SWSS_LOG_ENTER();
+
+    ++port.m_flap_count;
```

Comment:
looks like this will be incremented unnecessary if SAI has some flow where it notifies the same oper status multiple times?

Reply:
@prsunny no, there is a check where OA return if same port status is updated :- https://github.com/sonic-net/sonic-swss/blob/master/orchagent/portsorch.cpp#L7565